Content.Server/Atmos/Piping/Unary/Components/GasVentPumpComponent.cs

```diff
@@ -59,6 +59,48 @@ public sealed partial class GasVentPumpComponent : Component
         [DataField("underPressureLockoutLeaking")]
         public float UnderPressureLockoutLeaking = 0.0001f;
 
+        #region some averaging stuff
+        [ViewVariables(VVAccess.ReadOnly)]
+        public int samples { get; set; } = 0;
+
+        /// <summary>
+        ///     Calculate average pressure over X atmos updates.
+        /// </summary>
+        [ViewVariables(VVAccess.ReadOnly)]
+        public int maxSamples { get; set; } = 5;
```

Comment:
If this changes, it does not appear that the `measurements` or `pressurizationRate` ring buffer are resized correctly.

Reply:
I probably forgot to fix this issue after finding out that those arrays can't be initialized from the maxSamples variable, haven't figured out yet the proper way to do it. 
Perhaps a static variable could do it? Though I'm not sure how configurable that would be.